internal/gps/result.go

```diff
@@ -10,6 +10,10 @@ import (
 	"path/filepath"
 )
 
+// NumDepTreeWorkers determines the number of
+// WriteDepTree worker goroutines
+var NumDepTreeWorkers = 2
```

Comment:
yep! we haven't stabilized the API yet, so there's no obligation to keep it the same.